components/cam/bld/configure

```diff
@@ -958,11 +951,11 @@ my $clubb_sgs = $cfg_ref->get('clubb_sgs');
 
 # consistency checks...
 
-# CLUBB_SGS only works with mg or p3 microphysics
```

Comment:
just noting locations where shoc/p3 might be removed.

Reply:
we need to retain all lines and code blocks that have "shoc_sgs"